xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -484,3 +458,20 @@ function init() {
 // End XWiki augmentation.
 return XWiki;
 }(XWiki || {}));
+
+require(['jquery'], function ($) {
+  /**
+ * Permalink: Events triggered when the permalink modal is displayed
+ */
+  $(document).on('show.bs.modal','#permalinkModal', function (event) {
+    // Updating the permalink inside modal
+    var modal = $(this);
+    var button = $(event.relatedTarget);
+    var permalinkValue = button.attr('href');
```

Comment:
This gives you the value of the ``href`` attribute which is a relative URL. Shouldn't the permalink be an absolute URL?